copilot/environments/addons/ctime-app-cluster.yml

```
@@ -0,0 +1,149 @@
+      Description: !Ref 'AWS::StackName'
+      Family: 'aurora-mysql8.0'
+      Parameters:
+        character_set_client: 'utf8'
+  
+  ctimeappclusterDBCluster:
+    Metadata:
+      'aws:copilot:description': 'The ctimeappcluster Aurora Serverless v2 database cluster'
+    Type: 'AWS::RDS::DBCluster'
+    Properties:
+      MasterUsername:
+        !Join [ "",  [ '{{resolve:secretsmanager:', !Ref ctimeappclusterAuroraSecret, ":SecretString:username}}" ]]
```

YAMLlint (1.35.1)

[warning] too many spaces after comma

(commas)